### PR TITLE
Feat/generate sources add database and schema

### PR DESCRIPTION
**Pull request by jeremyholtzman**
resolves [#123](https://github.com/dbt-labs/dbt-codegen/issues/123)

This is a:
- [ ] documentation update
- [ ] bug fix with no breaking changes
- [x] new functionality
- [ ] a breaking change

## Checklist
- [ ] This code is associated with an issue which has been triaged and [accepted for development](https://docs.getdbt.com/docs/contributing/oss-expectations#pull-requests). 
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)
- [x] I have added an entry to CHANGELOG.md

### CODE REVIEW

**Review by dbeatty10 — CHANGES_REQUESTED** [review]
Now that we're aligned on the behavior described in https://github.com/dbt-labs/dbt-codegen/issues/123#issuecomment-1529895931, we can tackle all the implementation details 🚀 

Can you do each of the following before we dive any deeper into the code review?
1. Remove any code formatting not related to the actual implementation of the new feature
2. Use the following two (shorter) parameter names:
    - `include_database`
    - `include_schema`

### More detail
There appear to be a lot of unnecessary code formatting, so I didn't go through and make code suggestions for each. You'll be able to tell if you caught them all if the diff shown within the pull request is minimal.

File: CHANGELOG.md [hunk 1]
```diff
@@ -150,4 +151,4 @@ Bump utils version range
 Small quality of life improvements
 
 # dbt-codegen v0.0.1
-Initial release
+Initial release
```
Comment (dbeatty10) [inline CHANGELOG.md:154]: Is it possible to remove this code change as well?

It's not a huge deal, but I prefer to keep PRs as simple as humanly possible 😎

Comment (jeremyholtzman) [reply CHANGELOG.md:154]: Definitely, I must have missed that one!

**Review by dbeatty10 — APPROVED** [review]
Looks great @jeremyholtzman -- thanks for all your work on this!

**Pull request merged**